### PR TITLE
Components Overview documentation refinement

### PR DESCRIPTION
**Pull request by AnjiManova**
- Keywords for Figma and Adobe XD
- Adding Figma section
- Added the componentVersion as a structural layer in Figma

In another commit, I will upload an image for the Figma components overview

### CODE REVIEW

File: en/components/components-overview.md [hunk 1]
```diff
@@ -1,30 +1,40 @@
 ---
 title: Components Overview - Design System Components
 _description: The Indigo Design System contains 60+ UI Components with numerous presets, states, and elaborate built-in styling capabilities.
```
Comment (imincheva) [inline en/components/components-overview.md:3]: Is 60+ UI components correct? On the kit's cover it says 50+

File: en/components/components-overview.md [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
 
-<img class="responsive-img" src="../images/components-page-xd.png" />
+<img class="responsive-img" src="../images/components-page-xd.png" srcset="../images/components-page-xd@2x.png 2x" />
 
 > [!WARNING]
-> Detaching a component from the library in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
+> Detaching a component from the library in Sketch makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers containing metadata descriptions of the state, template, and other properties of the component. In Figma libraries, there is one additional layer placed in every component named `🚫 componentVersion` containing the version, which helps for successful code generation.
```
Comment (imincheva) [inline en/components/components-overview.md:37]: I think it's better to separate the info for Figma from the info for Sketch like it's done with Adobe XD.

**Review by imincheva — COMMENTED** [review]
Overall I think the Figma information is written well. I left two comments, which I'm not sure are relevant, so I'll leave it to @sdimchevski to decide whether the changes are necessary or not.

File: en/components/components-overview.md (hunk 2 above)
Comment (AnjiManova) [reply en/components/components-overview.md:37]: This is a warning box where is a summary for detaching. By default, it was for both Sketch and Adobe, and this is why I followed the same logic, but yes, I can split it somehow; just let me know which will be the optimal @sdimchevski

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**